### PR TITLE
Added new sampling method: krMCMC

### PR DESCRIPTION
**Pull request by SMANahian**

### CODE REVIEW

File: src/blant-sampling.c [hunk 1]
```diff
@@ -869,6 +869,93 @@ double SampleGraphletMCMC(GRAPH *G, SET *V, unsigned *Varray, int k, int whichCC
...
+	}
+    } else {
+	if(ocount < 0) {
+	    Warning("ocount (%g) is less than 0\n", ocount);
+	}
+	_graphletConcentration[GintOrdinal] += ocount;
+    }
+
+    _g_overcount = ocount;
+    return 1.0;
```
Comment (waynebhayes) [inline src/blant-sampling.c:956]: I think you want this to be "return ocount". That's the whole point of the double return value.

**Review by waynebhayes — CHANGES_REQUESTED** [review]
I think everything looks good except the "return 1.0". Please confirm if I'm right--I could be wrong... and resubmit the PR.

File: src/blant-sampling.c (hunk 1 above)
Comment (SMANahian) [reply src/blant-sampling.c:956]: I'm not sure. I saw this on the SampleGraphletMCMC function. 
The `SampleGraphletMCMC()` function also returns 1.0 everywhere.

The sampling functions are only called inside the `SampleGraphlet()` function where the return value aren't even being processed.

Comment (waynebhayes) [reply src/blant-sampling.c:956]: Ah. OK then. My *intent* was to return the desired weight, but maybe it's not being used anywhere... :-/

**Review by waynebhayes — APPROVED** [review]

**Pull request merged**